test/common/testimagetask.py

```diff
@@ -31,18 +31,20 @@ def start_publishing(self, host, github):
         body_text = ("Image creation for %s in process on %s.\nLog: :link"
                      % (self.image, testinfra.HOSTNAME))
 
+        # Update the body for an existing issue
         if self.issue:
             requests += [
                 # Get issue
                 { "method": "GET",
                   "resource": github.qualify("issues/" + str(self.issue['number'])),
                   "result": "issue"
                 },
-                # Add comment
+                # Update the issue
                 { "method": "POST",
-                  "resource": ":issue.comments_url",
+                    "resource": ":issue.url",
                   "data": {
-                      "body": body_text
+                      "body": body_text,
+                      "labels": [ "bot", "image/" + self.image ]
```

Comment:
Do these new labels need to be defined first, or are they being created on the fly with this request?